app/scripts/ui/color-button.js

```diff
@@ -17,7 +17,7 @@ function ColorButton(options, ui, drawingTool) {
   options.onClick = callback;
   BasicButton.call(this, options, ui, drawingTool);
 
-  this.setBackground(options.color);
+  // this.setBackground(options.color);
```

Comment:
Is this intentionally commented out?

Reply:
Yes - I need to clean this up as the color button now uses an svg icon per color instead of programatically setting the color.